docs/podman-pod-exists.1.md

```diff
@@ -0,0 +1,40 @@
+% PODMAN(1) Podman Man Pages
```

Comment:
I think I missed this in the other *-exists* man pages, but this should be 
% podman-pod-exists(1)
right?